### PR TITLE
[W5.7][T10-1]Pang Jia Hao

### PR DESCRIPTION
**Pull request by pangjiahao**
Add set tags functionality which modifies the tags of people in the addressbook.
Updated JUnit, I/O, and User guide

### CODE REVIEW

File: test/expected.txt [hunk 1]
```diff
@@ -138,6 +138,18 @@
...
+|| ===================================================
+|| Enter command: || [Command entered:  settags 1]
+|| New Tags: Adam Brown Phone: 111111 Email: adam@gmail.com Address: 111, alpha street Tags: 
+|| ===================================================
+|| Enter command: || [Command entered:  settags 2 t/friendly]
+|| New Tags: Betsy Choo Tags: [friendly]
+|| ===================================================
+|| Enter command: || [Command entered:  settags 2 t/secretive]
+|| New Tags: Betsy Choo Tags: [secretive]
+|| ===================================================
```
Comment (bryanwongweiheng) [inline test/expected.txt:152]: Perhaps you could add a test for multiple tags as i think users will often do that

File: src/seedu/addressbook/commands/SetTagCommand.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    public static final String COMMAND_WORD = "settags";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sets the tags of the person "
+            + "identified by the index number in the last shown person listing.\n"
+            + "Parameters: INDEX [t/TAG]\n"
+            + "Example: " + COMMAND_WORD + " 1 t/friend t/evil";
+
+    static final String MESSAGE_VIEW_NEW_TAGS = "New Tags: %1$s";
+
+    private Set<Tag> tags;
```
Comment (SleepySanjinLi) [inline src/seedu/addressbook/commands/SetTagCommand.java:27]: Does this means that it can add multi-tags?

**Review by SleepySanjinLi — COMMENTED** [review]
It's a very convenient method that we only need to add the tag to change the information stored without add the whole sentence.

File: test/expected.txt (hunk 1 above)
Comment (yunpengn) [reply test/expected.txt:152]: Good idea!

File: src/seedu/addressbook/commands/SetTagCommand.java (hunk 2 above)
Comment (yunpengn) [reply src/seedu/addressbook/commands/SetTagCommand.java:27]: Good idea!

File: test/java/seedu/addressbook/commands/SetTagCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+        SetTagCommand command = null;
+        try {
+            command = createSetTagCommand(targetVisibleIndex, tags, actualAddressBook, displayList);
+        } catch (IllegalValueException e) {
+            e.printStackTrace();
+        }
+        assertCommandBehaviour(command, expectedMessage, expectedAddressBook, actualAddressBook);
+    }
+}
```
Comment (yunpengn) [inline test/java/seedu/addressbook/commands/SetTagCommandTest.java:194]: Maybe you want to add a new line at the end of file (EOF).

**Review by yunpengn — APPROVED** [review]
This is a very useful feature. Please close this PR after viewing my comments.

File: test/java/seedu/addressbook/commands/SetTagCommandTest.java (hunk 3 above)
Comment (pangjiahao) [reply test/java/seedu/addressbook/commands/SetTagCommandTest.java:194]: Thanks for the comments!

**Pull request closed without merging**